development/cpp/introduction_to_godot_development.rst

```diff
@@ -37,4 +37,29 @@ Or:
 .. code:: bash
 
     $ gdb godot
-    > run -e -path ~/myproject
+    > run -e ~/myproject/project.godot
+
+(With the path pointing to the project.godot or pointing to a specific scene to open the editor at.)
```

Comment:
This is wrong, you cannot use the path to a scene directly. You can optionally put the scene as another argument, but not replacing the project path.